.scripts/build_steps.sh

```diff
@@ -55,3 +55,4 @@ find ~/staged-recipes-copy/recipes -mindepth 2 -maxdepth 2 -type f -name "yum_re
 python ${CI_SUPPORT}/build_all.py
 
 touch "/home/conda/staged-recipes/build_artifacts/conda-forge-build-done"
+sleep infinity
```

Comment:
Why are you trying to timeout the CI?

Reply:
Ooops ... I did not mean to push that. My mistake. Sorry again. It has been removed.